homeassistant/components/plex/sensor.py

```diff
@@ -139,6 +117,43 @@ def update(self):
         self._state = len(self.sessions)
         self._now_playing = now_playing
 
+        self.async_write_ha_state()
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return self._name
+
+    @property
+    def unique_id(self):
+        """Return the id of this plex client."""
+        return self._unique_id
+
+    @property
+    def should_poll(self):
+        """Return True if entity has to be polled for state."""
+        return False
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        return self._state
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit this state is expressed in."""
+        return "Watching"
+
+    @property
+    def icon(self):
+        """Return the icon of the sensor."""
+        return "mdi:plex"
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        return {content[0]: content[1] for content in self._now_playing}
```

Comment:
We normally want to set the state attributes keys explicitly to be able to check formatting and content.

Reply:
This was legacy code copied over as-is. It's worked for a long time, I'd rather not touch it right now.